lib/std/os.zig

```diff
@@ -3734,6 +3734,7 @@ pub fn faccessatW(dirfd: fd_t, sub_path_w: [*:0]const u16, mode: u32, flags: u32
         .SUCCESS => return,
         .OBJECT_NAME_NOT_FOUND => return error.FileNotFound,
         .OBJECT_PATH_NOT_FOUND => return error.FileNotFound,
+        .OBJECT_NAME_INVALID => return error.BadPathName,
```

Comment:
Shouldn't this be `unreachable`?

Reply:
Yes, this is probably more correct, thanks.